package.json

```diff
@@ -6,6 +6,7 @@
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@fortawesome/react-fontawesome": "^0.2.0",
+    "@mui/base": "^5.0.0-beta.40",
```

Comment:
This package added but not used, could you remove this ?

Reply:
While setting up the project locally, I encountered with this error. That's why the mentor suggested few solutions and installing this package resolved the error. This was the error:
![mui error](https://github.com/user-attachments/assets/ea4df1f0-b8a6-4e8b-bc05-899dbe679153)

Reply:
Okay fine, @harmeetsingh11. 
Then could you also add package-lock.json ?

Reply:
I have used `bun` to install packages and run the project. It generates `bun.lockb` file. `Bun` installs packages, starts and run faster than `npm`. There is no need for package-lock.json.

Reply:
This project is using `package-lock.json` which is used to pick specific version to install for the CI tool in this projects context `vercel`

Reply:
ok. I encountered several errors while running the project locally using `npm`. 
So, I can remove the newly installed package `@mui/base` from the `package.json` file. I think the deployed project should work fine because their will be no change in `package.json` and `package-lock.json` files.
![error](https://github.com/user-attachments/assets/21919dc4-6484-4671-aa9f-16dd379427b1)

Reply:
Sure, let's go with that.